docs/conf.py

```diff
@@ -11,10 +11,15 @@
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
+import logging
 import shutil
 import sys, os, pkg_resources, datetime
+from distutils.version import Version
```

Comment:
`distutils` has been deprecated. We should use `packaging.version` instead.

Reply:
This is not actually being used, I'll remove it. Should I still change the call to `pkg_resources` to use `packaging` instead ?

Reply:
That would be good, though I think you're looking for `importlib.metadata.version`.